examples/dynamo/cross_runtime_compilation_for_windows.py

```diff
@@ -0,0 +1,80 @@
+"""
+.. _resnet_cross_runtime_compilation_for_windows_example:
+
+cross runtime compilation limitations:
+The cross compile and saved model can only be loaded in Windows, it can no longer be loaded in Linux
+The cross compile and saved model can only be loaded in the same Compute capacity as the Linux which it was cross compiled
```

Comment:
I think the formal term in CUDA is [compute capability](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capability) rather than `compute capacity`.